### PR TITLE
Show icons on primary button in filled variant

### PR DESCRIPTION
**Pull request by yepzdk**
https://platform.dandigbib.org/issues/5391

This pr adresses the posibility of having the external-link button in a filled variant. 
To achieve this a new icon component for IconExternalLink is added. Also the svg path fill and stroke has been set to currentColor, to get the ability of inheriting the color property and changing it from css. This also applies for arrow small right, since it can too be a part of a filled button. 
We might consider making currentColor default for all svgs.

Final result:
Screenshot 2022-08-08 at 14 16 54

### CODE REVIEW

File: src/stories/Library/Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <svg width="61" height="9" viewBox="0 0 61 9" fill="none" xmlns="http://www.w3.org/2000/svg">
-    <path className="arrow__body" d="M60 4.5H0" stroke="black"/>
-    <path className="arrow__head" d="M60.3537 4.85355C60.5489 4.65829 60.5489 4.34171 60.3537 4.14645L57.1717 0.96447C56.9764 0.769208 56.6598 0.769208 56.4646 0.96447C56.2693 1.15973 56.2693 1.47631 56.4646 1.67157L59.293 4.5L56.4646 7.32843C56.2693 7.52369 56.2693 7.84027 56.4646 8.03553C56.6598 8.2308 56.9764 8.2308 57.1717 8.03553L60.3537 4.85355ZM60.0001 4H57.0001V5H60.0001V4Z" fill="black"/>
+    <path className="arrow__body" d="M60 4.5H0" stroke="currentColor"/>
```
Comment (kasperg) [inline src/stories/Library/Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg:2]: :+1: This is a nice improvement and something we should consider for all of our SVGs.

**Review by sinejespersen — APPROVED** [review]

File: src/stories/Library/Buttons/button/buttons.scss [hunk 2]
```diff
@@ -65,6 +65,10 @@ $c-btn-border-disabled: $c-global-tertiary-2;
 .btn-filled {
   background-color: $c-text-primary-black;
   color: $c-text-primary-white;
+
+  .icon {
+    color: $c-text-primary-white;
+  }
```
[the remark this answers is not in the document]
Comment (yepzdk) [reply src/stories/Library/Buttons/button/buttons.scss]: You are right. Thanks.
I have instead made color inherit on the existing btn-icon class.

File: src/stories/Library/Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg (hunk 1 above)
Comment (yepzdk) [reply src/stories/Library/Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg:2]: Thanks. I'm glad that you agree.
We will address this in a separate PR.

**Review by kasperg — APPROVED** [review]
:+1: LGTM

**Pull request merged**